src/components/presentationals/Button/Button.jsx

```diff
@@ -5,18 +5,23 @@ import './Button.scss';
 
 const Button = props => {
   return (
-    <button className={`btn ${props.button.class}`} type={props.button.class}>
+    <button
```

Comment:
This button component can be rewritten to 
```javascript 
const Button = props => ( 
<button 
    className={`btn ${props.className}`}
    type={props.className}
    onClick={props.onClick}
>
   {props.value}
</button>

Reply:
Done